src/Microsoft.TemplateEngine.TemplateLocalizer.Core/TemplateLocalizer.cs

```diff
@@ -30,21 +32,36 @@ public TemplateLocalizer(ILoggerFactory? loggerFactory)
 
         public async Task<ExportResult> ExportLocalizationFilesAsync(string templateJsonPath, ExportOptions options, CancellationToken cancellationToken = default)
         {
-            using FileStream fileStream = new FileStream(templateJsonPath, FileMode.Open, FileAccess.Read);
-
             JsonDocumentOptions jsonOptions = new JsonDocumentOptions()
             {
                 CommentHandling = JsonCommentHandling.Skip,
                 AllowTrailingCommas = true,
             };
-            using JsonDocument jsonDocument = await JsonDocument.ParseAsync(fileStream, jsonOptions, cancellationToken).ConfigureAwait(false);
 
-            TemplateStringExtractor stringExtractor = new TemplateStringExtractor(jsonDocument, _loggerFactory);
-            var strings = stringExtractor.ExtractStrings();
+            try
+            {
+                using FileStream fileStream = new FileStream(templateJsonPath, FileMode.Open, FileAccess.Read);
+                using JsonDocument jsonDocument = await JsonDocument.ParseAsync(fileStream, jsonOptions, cancellationToken).ConfigureAwait(false);
+
+                TemplateStringExtractor stringExtractor = new TemplateStringExtractor(jsonDocument, _loggerFactory);
+                IReadOnlyList<TemplateString> templateJsonStrings = stringExtractor.ExtractStrings();
 
-            // This section is not implemented yet and will be delivered in the future commits. Please ignore during review.
+                string targetDirectory = options.TargetDirectory ?? Path.Combine(Path.GetDirectoryName(templateJsonPath) ?? string.Empty, "localize");
 
-            return new ExportResult(templateJsonPath, "Operation Failed", null);
+                await TemplateStringUpdater.UpdateStringsAsync(
+                    templateJsonStrings,
+                    templateJsonLanguage: "en",
```

Comment:
In theory, in future we might want to add `--input-language` option if I understand correctly?

Reply:
That's the idea, yes.

Reply:
I think on call later, we came to conclusion, that it might make more sense, to have this defined inside `template.json` itself, and default would be `en`